### ISSUE TITLE
Limiting the maximum number of rooms

### ISSUE THREAD

**Issue by Tecumtha** [blindsidenetworks/moodle-mod_bigbluebuttonbn#298]
Is there a way to limit the number of rooms that can be opened in the Moodle Instanz.

We have the problem that the participants can be limited to 40 people per room, but the number of rooms is not limited.

**Comment by jfederico** [comment]
That is not possible from the plugin.

Some commercial solutions have the feature implemented through the Loadbalancer or Broker. That is how we do it.

**Comment by Corradones** [comment]
Hello jfederico, can you pleqse inform me which loadbalancer is used?I'm interested in building up this solution. Is there some project bbb-oreinted in github?
Really thanks

Corrado

**Comment by ak4t0sh** [comment]
@Corradones https://github.com/blindsidenetworks/scalelite ;)

**Comment by Corradones** [comment]
ok, i thought was some other one. I know this one already, thanks anyway :-)

**Comment by jfederico** [comment]
No, Scaleite does not support that either and it is not planned to do so. You could extend it, but that would be all on you.

Just be aware of Scalelite is released under AGPL. You are forced to keep your modifications public even if you are only hosting.

**Comment by ak4t0sh** [comment]
oh sorry your right @jfederico I thought scalelite implemented it ! my bad
